hack/update-codegen-crds.sh

```diff
@@ -17,15 +17,13 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-set -o xtrace 
+# set -o xtrace
```

Comment:
TODO?

Reply:
or remove. No need for xtrace in our hack tools.